src/components/NewGamePage/NewGamePage.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components/macro'
+import AddPlayerForm from '../AddPlayerForm'
+import Button from '../Button'
+import { NavLink } from 'react-router-dom'
+
+export default function NewGamePage({ addPlayer, players, resetForm, name }) {
+  return (
+    <>
+      <AddPlayerForm
+        key={name}
+        role="form"
+        onAddPlayer={addPlayer}
+        disabled={players.length >= 4}
+        placeholderText={
+          players.length >= 4
+            ? 'Your flight has reached the max. of 4 players'
+            : 'Enter player name (e.g. Joe)'
+        }
```

Comment:
Great idea to limit the number of players and set the input field to "disabled" so it fits. Good work! <3